src/utils.js

```diff
@@ -36,6 +36,38 @@ function getRankingHeadingElement(heading) {
 	return null;
 }
 
+function getText(element) {
+	// is a TEXT_NODE or CDATA section
+	if (element.nodeType == 3 || element.nodeType == 4) {
+		return element.nodeValue;
+	}
+
+	if (element.nodeType != 1) {
+		return '';
+	}
+
+	// tag with alt attribute
+	if (getTagName(element) == 'IMG' || (getTagName(element) == 'INPUT' && element.getAttribute('type').toLowerCase() == 'image')) {
+		var alternatives = '';
+		if (element.getAttribute('alt')) {
+			alternatives += element.getAttribute('alt');
+		}
+		return alternatives;
+	}
+
+	var texto = [];
+
+	// get text from child nodes
```

Comment:
Actually it is covered. Because, when I comment this part out, tests will fail.

I'll extend the html test to be more explicit about nested childs.